src/components/Button/Button.test.tsx

```diff
@@ -0,0 +1,118 @@
+import userEvent from '@testing-library/user-event';
+import { describe, it, expect, vi } from 'vitest';
+
+import { render, screen } from '../../test/test-utils';
+
+import { Button } from './Button';
+
+describe('Button', () => {
+  it('renderiza con props por defecto', () => {
```

Comment:
Test descriptions (it blocks) are written in Spanish (e.g. this line). The project enforces English for code, tests and documentation. This occurs multiple times in the file (lines 9, 18, 29, 42, 57, 69, 81, 93, 104, 111).

**Sugerencia:**
```typescript
Translate all test descriptions to English. For example: `it('renders with default props', () => { ... })` and `it('calls onClick when enabled', async () => { ... })`.
```